niceman/distributions/debian.py

```diff
@@ -160,10 +161,49 @@ def install_packages(self, session, use_version=True):
         lgr.debug("Installing %s", ', '.join(package_specs))
         session.execute_command(
             # TODO: Pull env out of provenance for this command.
-            ['apt-get', 'install', '-y'] + package_specs,
+            ['apt-get', 'install', '-y', '--force-yes'] + package_specs,
```

Comment:
from TFM:
```
       --force-yes
           Force yes; this is a dangerous option that will cause apt to
           continue without prompting if it is doing something potentially
           harmful. It should not be used except in very special situations.
           Using force-yes can potentially destroy your system! Configuration
           Item: APT::Get::force-yes. This is deprecated and replaced by
           --allow-unauthenticated , --allow-downgrades ,
           --allow-remove-essential , --allow-change-held-packages in 1.1.
```
so do we just need `--allow-downgrades`?

Reply:
Yes, that's a better choice and it works. Making this change.